egs2/minds14/asr1/conf/smaller_conformer_wav2vec.yaml

```diff
@@ -0,0 +1,89 @@
+specaug: specaug
```

Comment:
It is cool to have many configs, but it seems that many of the configs are very similar to each other, could you try to combine them into fewer settings (e.g., the one without SSL features, the one with SSL features)